src/vrv.cpp

```diff
@@ -84,37 +84,45 @@ bool Resources::InitFonts()
         return false;
     }
 
+    if (!SetTextFont("VerovioText-1.0", false)) {
```

Comment:
With current implementation Verovio would search for italic/bold/etc. fonts for VerovioText which we don't have and can't have (since it's smufl font).